### PR TITLE
feature: Support TensorFlow 2.9.1

### PR DESCRIPTION
**Pull request by navinsoni**
- Upgrade TF2.8.0 to TF2.9.1

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -97,6 +97,10 @@ Please refer to below table for release support information:
      - Sagemaker TensorFlow Extensions Release Version
      - TensorFlow Release Version
      - Sagemaker TensorFlow Extentions Supported Python Versions
+   * - 2.9.0.1.15.x
+     - v1.15.x
+     - 2.9.0
```
Comment (nish21) [inline README.rst]: Tensorflow released 2.9.1 with a bug fix yesterday. Does this need to be updated?

**Review by nish21 — COMMENTED** [review]
LGTM otherwise

File: README.rst (hunk 1 above)
Comment (navinsoni) [reply README.rst]: in that case we can release extension with 2.9.1 rather than 2.9.0

Comment (navinsoni) [reply README.rst]: updated to 2.9.1 due to bug fix

**Review by nish21 — APPROVED** [review]

**Review by shreyapandit — APPROVED** [review]

**Pull request merged**